folder/README.md

```diff
@@ -6,4 +6,27 @@ We are about to make a commit without best practices.
 
 hi
 
-goodbye
\ No newline at end of file
+goodbye
+
+docs
+
+
+repositories, branches, commits, tags
+
+# created the repo locally
+local repository -> "remote" repository (git add remote <name> <URL>)
+
+# created the repo remotely
+git clone <URL>
+
+# create a branch locally
+git checkout <branch>
+git checkout -b <new branch>
+git add ...; git commit -m "..." (several times)
+git push --set-upstream origin <new branch>
+git pull [origin <new branch>]
+
+# create a new branch remotely
+using the GitHub UI to create <branch>
+git fetch --all
+git checkout <branch>
```

Comment:
I find this section vague. Please consider adding a verbal description of what these commands are actually doing.